test/new-e2e/pkg/utils/e2e/client/agentclientparams/agent_client_params.go

```diff
@@ -43,3 +45,10 @@ func WithSkipWaitForAgentReady() Option {
 		p.ShouldWaitForReady = false
 	}
 }
+
+// WithAgentInstallPath sets the agent installation path
+func WithAgentInstallPath(path string) Option {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this possible on Linux?

Reply:
not to my knowledge